versions.md

```diff
@@ -1,6 +1,8 @@
 Forthcoming
 -----------
 
+* Added participant property to configure SHM transport metatraffic behavior.
```

Comment:
I'd point here that the default behaviour is no metatraffic over SHM

Reply:
932a0dc